src/net.h

```diff
@@ -650,6 +651,7 @@ class CNode
     const NodeId id;
 
     const uint64_t nKeyedNetGroup;
+    std::atomic_bool fPauseRecv;
```

Comment:
This is never set to true until a later commit. I'm generally ok with this, but it is a bit strange.